src/import-data/ImportData.ts

```diff
@@ -75,7 +75,7 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
         jobId,
         wdsProxyUrl: wdsUrl,
       };
-      asyncImportJobStore.update(Utils.append(newJob));
+      asyncImportJobStore.update(Utils.append<AsyncImportJob>(newJob));
```

Comment:
A type parameter is necessary here because `newJob` is an `AzureAsyncImportJob`, but we're appending to a more general list of `AsyncImportJob[]`.